code/game/objects/structures/tables_racks.dm

```diff
@@ -35,6 +35,29 @@
 	var/deconstruction_ready = TRUE
 	var/flipped = 0
 
+/obj/structure/table/Bumped(mob/living/carbon/human/H)
```

Comment:
Not sure if typing this is a good idea. It sorta implies that only humans can call it, which is not true.